packages/remix-react/components.tsx

```diff
@@ -1060,6 +1060,18 @@ export function useFetcher<TData = AppData>(
   return useFetcherRR(opts);
 }
 
+export const DevScripts =
+  process.env.NODE_ENV === "production"
```

Comment:
@pcattori Wouldn't it be better to check for `process.env.NODE_ENV === "development"` and anything else than that should return null?

Maybe someone runs a NODE_ENV like `staging` or `preview` but in those cases, you wouldn't have HMR active.